lib/elixir/lib/module/types/descr.ex

```diff
@@ -62,7 +62,10 @@ defmodule Module.Types.Descr do
   def pid(), do: %{bitmap: @bit_pid}
   def port(), do: %{bitmap: @bit_port}
   def reference(), do: %{bitmap: @bit_reference}
-  def tuple(), do: %{bitmap: @bit_tuple}
+  def open_tuple(elements), do: %{tuple: tuple_new(:open, elements)}
+  def tuple(elements), do: %{tuple: tuple_new(:closed, elements)}
+  def empty_tuple(), do: tuple([])
```

Comment:
Nitpick: I would remove this function. Empty tuples are very rare and, opposite to `empty_map`, we don't emit it on `to_quoted`.